rpc/send_test.go

```diff
@@ -1,34 +1,34 @@
-// Copyright 2015 The Cockroach Authors.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
-// implied. See the License for the specific language governing
-// permissions and limitations under the License. See the AUTHORS file
-// for names of contributors.
-//
-// Author: joezxy (joe.zxy@foxmail.com)
-
-package rpc
-
-import (
-	"testing"
-)
-
-func TestInvalidAddrLength(t *testing.T) {
-
-	// The provided addrs is nil, so its length will be always
-	// less than the specified response number
-	ret, err := Send(Options{N: 1}, "", nil, nil, nil, nil)
-
-	// the expected return is nil and SendError
-	if _, ok := err.(SendError); !ok || ret != nil {
-		t.Fatalf("Shorter addrs should return nil and SendError.")
-	}
-}
+// Copyright 2015 The Cockroach Authors.
```

Comment:
This file had DOS-style line endings (CRLF) before; your change fixed it to have unix-style line endings (LF). That's what we want, but github gets confused by this and shows the entire file as being rewritten.